static/js/schema-impl/capability/thing.js

```diff
@@ -300,13 +294,7 @@ class Thing {
         for (const name in description.actions) {
           const action = description.actions[name];
 
-          let href;
-          for (const form of description.actions[name].forms) {
-            if (!form.op || form.op === Constants.WoTOperation.INVOKE_ACTION) {
-              href = form.href;
-              break;
-            }
-          }
+          const href = Utils.selectFormHref(action.forms, Constants.WoTOperation.INVOKE_ACTION);
```

Comment:
For consistency, I always used `selectFormHref` when looking for the gateway form.